src/app/layout.jsx

```
@@ -4,63 +4,62 @@ import HeaderWrapper from './HeaderWrapper';
     var d = document.documentElement;
-    var c = document.cookie.split('; ').find(function (r) { return r.indexOf('theme=') === 0; });
-    var v = c ? decodeURIComponent(c.split('=')[1]) : '';
-    var prefersDark = window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches;
-    var dark = v ? (v === 'dark') : prefersDark;
-    d.classList.toggle('dark', !!dark);
-  } catch (e) {}
-})();
-        `}</Script>
+    if (!d.classList.length) {
+      var c = document.cookie.split('; ').find(function (r) { return r.indexOf('theme=') === 0; });
+      var v = c ? decodeURIComponent(c.split('=')[1]) : '';
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)